### DISCUSSION TITLE
Jira integration steps

### DISCUSSION THREAD

**Discussion by rsandidfc** [bsorrentino/maven-confluence-plugin#274] [Q&A] [answered]
Hi all - I've done a simple Jira integration. It works great! My question is, is there a hook in the plugin to filter issues on a specific Jira component (as defined in Jira) because the Jira project has many components. In my template I have:

```
h1. Jira
${gitlog.sinceTagName}
${gitlog.jiraIssues}
```

Thanks!

**Comment by bsorrentino** [comment, marked as answer]
Hi @rsandidfc 

Jira integration is an pretty old feature integrated in the `2015` from a PR 

I've created a dedicated module `gitlog+jira` to facilitate its enhancement

in any case  feature that you need it is not implemented yet
